packages/mobile/src/screens/chat-screen/ChatUserListScreen.tsx

```diff
@@ -0,0 +1,274 @@
+  },
+  loadingSpinner: {
+    height: spacing(20),
+    width: spacing(20),
+    alignSelf: 'center'
+  },
+  searchContainer: {
+    marginTop: spacing(8),
+    marginHorizontal: spacing(2),
+    marginBottom: spacing(2)
+  },
+  searchIcon: {
+    width: spacing(5),
+    height: spacing(5)
+  },
+  searchInputContainer: {
+    paddingRight: spacing(4.5),
+    paddingVertical: spacing(4.5)
+  },
+  searchInputText: {
+    fontWeight: '700',
+    fontSize: typography.fontSize.large
+  },
+  profilePicture: {
+    height: spacing(18),
+    width: spacing(18)
+  },
+  userContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    padding: spacing(4),
+    borderBottomColor: palette.neutralLight4,
+    borderBottomWidth: 1
+  },
+  userNameContainer: {
+    flexGrow: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    marginLeft: spacing(2.5)
+  },
+  userName: {
+    fontSize: typography.fontSize.small,
+    fontWeight: 'bold',
+    color: palette.neutral
+  },
+  followContainer: {
+    marginTop: spacing(1),
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between'
+  },
+  handle: {
+    fontSize: typography.fontSize.small,
+    color: palette.neutral
+  },
+  followersContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  followersCount: {
+    fontWeight: '800',
+    marginHorizontal: spacing(1),
+    color: palette.neutralLight4,
+    fontSize: typography.fontSize.small
+  },
+  followers: {
+    color: palette.neutralLight4,
+    fontSize: typography.fontSize.small
+  },
+  iconUser: {
+    height: spacing(4),
+    width: spacing(4)
+  },
+  followsYouTag: {
+    fontSize: typography.fontSize.xxs,
+    fontWeight: '900',
+    letterSpacing: 0.64,
+    textTransform: 'uppercase',
+    color: palette.neutralLight4,
+    borderWidth: 1,
+    borderRadius: spacing(1),
+    borderColor: palette.neutralLight4,
+    paddingVertical: spacing(1),
+    paddingHorizontal: spacing(2)
+  }
+}))
+
+type SearchUsersScreenProps = {
+  debounceMs?: number
+  defaultUserList?: {
+    userIds: ID[]
+    loadMore: () => void
+    loading: boolean
+  }
+}
+
+export const ChatUserListScreen = (props: SearchUsersScreenProps) => {
+  const {
+    debounceMs = DEBOUNCE_MS,
+    defaultUserList = {
+      userIds: [],
+      loading: false,
+      loadMore: () => {}
+    }
+  } = props
+  const styles = useStyles()
+  const palette = useThemeColors()
+  const [query, setQuery] = useState('')
+  const [hasQuery, setHasQuery] = useState(false)
+  const navigation = useNavigation<AppTabScreenParamList>()
+  const dispatch = useDispatch()
+
+  const { userIds, status } = useSelector(getUserList)
+  const users = useProxySelector(
+    (state) => {
+      const ids = hasQuery ? userIds : defaultUserList.userIds
+      const users = getUsers(state, { ids })
+      return ids.map((id) => users[id])
+    },
+    [hasQuery, userIds]
```

Comment:
I made some changes here recently on web - I think the proxy was to restrictive and wasn't getting mutuals on initial load. Not sure if relevant here!